### DISCUSSION TITLE
replacement of `read-pkg-up` to `read-package-up` not resulting in PR

### DISCUSSION THREAD

**Discussion by travi** [renovatebot/renovate#26469] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

i opened https://github.com/renovatebot/renovate/pull/26101 as a result of a [user report in semantic-release](https://github.com/semantic-release/semantic-release/issues/3085). i left the user report open, but havent come back to it since i know there is a delay between a change getting merged and being deployed to the hosted instance. had a [separate user report](https://github.com/semantic-release/semantic-release/pull/3127) come in today that reminded me about that switch and made me realize that i hadn't gotten a replacement PR.

our [org-level config](https://github.com/semantic-release/.github/blob/main/renovate-config.json) [extends `config:best-practices`](https://github.com/semantic-release/.github/blob/0e5f88716a05b6745ebe8b4599483acff07de406/renovate-config.json#L3), so it [should include `replacements:all`](https://docs.renovatebot.com/presets-config/#configrecommended)

the latest version of `read-pkg-up` and the only available version of `read-package-up` are both v11.0.0. could that be related to the problem?

minimal reproduction: https://github.com/travi/renovate-reproduction-read-pkg-up

### Relevant debug logs

<details><summary>Logs</summary>

the replacement was added in v37.83.0, so the current version is newer:
```

INFO: Repository started
{
  "renovateVersion": "37.103.1"
}

```

```
{
            "depType": "dependencies",
            "depName": "read-pkg-up",
            "currentValue": "^11.0.0",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "lockedVersion": "11.0.0",
            "updates": [],
            "packageName": "read-pkg-up",
            "versioning": "npm",
            "warnings": [],
            "deprecationMessage": "On registry `https://registry.npmjs.org`, the \"latest\" version of dependency `read-pkg-up` has the following deprecation notice:\n\n`Renamed to read-package-up`\n\nMarking the latest version of an npm package as deprecated results in the entire package being considered deprecated, so contact the package author you think this is a mistake.",
            "sourceUrl": "https://github.com/sindresorhus/read-pkg-up",
            "registryUrl": "https://registry.npmjs.org",
            "currentVersion": "11.0.0",
            "fixedVersion": "11.0.0"
          },
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by wwuck** [comment, marked as answer]
You missed adding the replacement to `replacements:all`.

https://github.com/renovatebot/renovate/pull/26667

**Reply by travi** [reply]
very interesting. thanks for sharing here. i always assumed that `replacements:all` took care of inclusion automatically, so i'll have to remember this for future additions
